From 936c3ea007b8a222c186696442bac4ad5e052d38 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Sun, 10 Mar 2024 18:12:31 +0100
Subject: [PATCH 07/11] scripts/buildflags.mk: sort the flag list

in order to ease the comparison with the output of dpkg-buildflags.
---
 scripts/mk/buildflags.mk | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/scripts/mk/buildflags.mk b/scripts/mk/buildflags.mk
index 700d2a1f7..7d949bc04 100644
--- a/scripts/mk/buildflags.mk
+++ b/scripts/mk/buildflags.mk
@@ -5,13 +5,13 @@
 #   CFLAGS: flags for the host C compiler.
 #   CPPFLAGS: flags for the host C preprocessor.
 #   CXXFLAGS: flags for the host C++ compiler.
-#   OBJCFLAGS: flags for the host Objective C compiler.
-#   OBJCXXFLAGS: flags for the host Objective C++ compiler.
 #   DFLAGS: flags for the host D compiler.
-#   FFLAGS: flags for the host Fortran 77 compiler.
 #   FCFLAGS: flags for the host Fortran 9x compiler.
+#   FFLAGS: flags for the host Fortran 77 compiler.
 #   GCJFLAGS: flags for the host java compiler.
 #   LDFLAGS: flags for the host linker.
+#   OBJCFLAGS: flags for the host Objective C compiler.
+#   OBJCXXFLAGS: flags for the host Objective C++ compiler.
 #
 # And the following variables for the build tools (since dpkg 1.22.1):
 #
@@ -19,13 +19,13 @@
 #   CFLAGS_FOR_BUILD: flags for the build C compiler.
 #   CPPFLAGS_FOR_BUILD: flags for the build C preprocessor.
 #   CXXFLAGS_FOR_BUILD: flags for the build C++ compiler.
-#   OBJCFLAGS_FOR_BUILD: flags for the build Objective C compiler.
-#   OBJCXXFLAGS_FOR_BUILD: flags for the build Objective C++ compiler.
 #   DFLAGS_FOR_BUILD: flags for the build D compiler.
-#   FFLAGS_FOR_BUILD: flags for the build Fortran 77 compiler.
 #   FCFLAGS_FOR_BUILD: flags for the build Fortran 9x compiler.
+#   FFLAGS_FOR_BUILD: flags for the build Fortran 77 compiler.
 #   GCJFLAGS_FOR_BUILD: flags for the host java compiler.
 #   LDFLAGS_FOR_BUILD: flags for the build linker.
+#   OBJCFLAGS_FOR_BUILD: flags for the build Objective C compiler.
+#   OBJCXXFLAGS_FOR_BUILD: flags for the build Objective C++ compiler.
 #
 # You can also export them in the environment by setting
 # DPKG_EXPORT_BUILDFLAGS to a non-empty value.
@@ -44,13 +44,13 @@ DPKG_BUILDFLAGS_LIST := $(foreach var,\
   CFLAGS \
   CPPFLAGS \
   CXXFLAGS \
-  OBJCFLAGS \
-  OBJCXXFLAGS \
   DFLAGS \
-  FFLAGS \
   FCFLAGS \
+  FFLAGS \
   GCJFLAGS \
   LDFLAGS \
+  OBJCFLAGS \
+  OBJCXXFLAGS \
   ,$(var) $(var)_FOR_BUILD)
 
 define dpkg_buildflags_export_envvar
-- 
2.39.2

